tests/limiter/limiter.js

```diff
@@ -74,28 +75,28 @@ describe('tests/limiter/limiter.js >', () => {
     expect(headers['x-ratelimit-remaining']).to.not.exist;
   }
 
-  describe('limit functionality >', () => {
-    afterEach((done) => setTimeout(done, 100));
-
-    function makeRequest(path, method, token = token1, data) {
-      if (!data && path === '/v1/aspects' && method === 'post') {
-        data = {name: `${tu.namePrefix}Limiter${i}`, timeout: '1m'};
-        i++;
-      }
+  function makeRequest(path, method, token = token1, data) {
+    if (!data && path === '/v1/aspects' && method === 'post') {
+      data = {name: `${tu.namePrefix}Limiter${i}`, timeout: '1m'};
+      i++;
+    }
 
-      return new Promise((resolve, reject) => {
-        api[method](path)
-        .set('Authorization', token)
-        .send(data)
-        .end((err, res) => {
-          if (err) {
-            reject(err)
-          } else {
-            resolve(res);
-          }
-        });
+    return new Promise((resolve, reject) => {
+      api[method](path)
+      .set('Authorization', token)
+      .send(data)
+      .end((err, res) => {
+        if (err) {
+          reject(err)
+        } else {
+          resolve(res);
+        }
       });
-    }
+    });
+  }
+
+  describe('limit functionality >', () => {
+    afterEach((done) => setTimeout(done, 100));
```

Comment:
must be careful with timeouts -- don't want to set it higher than necessary, but it might work ok locally and fail in travis

Reply:
Does travis normally require higher timeouts than local testing? Is there some metric to use or is it experimental with how long to set them

Reply:
trial and error :(